packages/html-reporter/src/reportView.tsx

```diff
@@ -23,10 +23,12 @@ import { HeaderView } from './headerView';
 import { Route } from './links';
 import type { LoadedReport } from './loadedReport';
 import './reportView.css';
+import type { GitCommitInfo } from './metadataView';
```

Comment:
So this is inside HTML report, so it can't know user has Git.

Reply:
Correct—one of the tests covers the case they either (1) don't have info/git, (2) manually set metadata to something that conflicts with our schema.